src/project:kbc/Login.java

```diff
@@ -0,0 +1,85 @@
+import javafx.application.Application;
+import javafx.scene.control.ComboBox;
+import javafx.collections.ObservableList;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.Scene;
+import javafx.scene.text.Text;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.control.Button;
+import javafx.scene.layout.FlowPane;
+import javafx.geometry.Pos;
+import javafx.event.ActionEvent;
+
+public class Login extends Application{
+	Scene scene1,scene2,scene3;
+	private TextField usernameField, passwordField, nameField, streetField, cityField, postCodeField, phoneField, newUsernameField, newPasswordField, rePasswordField;
+	public void start(Stage primaryStage){
+		primaryStage.setTitle("Login Page");
+		
+		Button userLoginButton = new Button("User Login");
+		Button librarianLoginButton = new Button("Librarian Login");
```

Comment:
We should only need a generic user Login button, once the login is passed it will check to see if it is a librarian or not.